int16 centroidX
int16 centroidY
sensor_msgs/Image mask
sensor_msgs/Image result
